test/metabase/lib/stage_test.cljc

```diff
@@ -178,7 +178,22 @@
                   :display-name             "Count"
                   :base-type                :type/Integer
                   :lib/source               :source/card
-                  :lib/desired-column-alias "count"}]
+                  :lib/desired-column-alias "count"}
+                 {:name                     "ID"
+                  :display-name             "ID"
+                  :base-type                :type/BigInteger
+                  :lib/source               :source/implicitly-joinable
+                  :lib/desired-column-alias "USERS__via__USER_ID__ID"}
+                 {:name                     "NAME"
+                  :display-name             "Name"
+                  :base-type                :type/Text
+                  :lib/source               :source/implicitly-joinable
+                  :lib/desired-column-alias "USERS__via__USER_ID__NAME"}
+                 {:name                     "LAST_LOGIN"
+                  :display-name             "Last Login"
+                  :base-type                :type/DateTime
+                  :lib/source               :source/implicitly-joinable
+                  :lib/desired-column-alias "USERS__via__USER_ID__LAST_LOGIN"}]
                 (lib/visible-columns query)))))))
```

Comment:
@camsaul Reverting this test is questionable, but required for this PR to work. In this case at least I know that issue I fixed in #30950 by removing implicitly joinable fields everywhere is fixed in this PR differently (i.e., only for orderable and breakoutable columns).

Reply:
Can't you just update the call to `visible-columns` to include the new option to exclude implicitly joinable columns for source Cards? Actually it would be even better if you tested both scenarios (with the new option enabled = include the implicitly joinable columns; with the new option disabled = exclude the columns)

Reply:
That can be done too.